model/file.go

```diff
@@ -0,0 +1,158 @@
+package model
+
+import (
+	"errors"
+	"github.com/unidoc/unipdf/v3/common"
+	"github.com/unidoc/unipdf/v3/core"
+)
+
+// (Section 7.11.3 p. 102).
+// See Table 44 - Entries in a file specification dictionary
+
+// PdfFilespec represents a file specification
```

Comment:
Would be nice to have a few words about what a "file specification" is, what is the use case?

Reply:
Is this mostly for specifying files via URLs?  Not embedded files etc?